### PR TITLE
feat: added configurable board title

### PR DESCRIPTION
**Pull request by ukiesmack**

### CODE REVIEW

**Review by haradzienski — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -25,9 +25,15 @@ const redisConfig = {
 	},
 };
 
+const options = {
+	uiConfig: {
+		boardTitle: config.BOARD_TITLE,
+	}
+}
```
Comment (ukiesmack) [inline src/index.js:32]:
```suggestion
};

```

File: src/index.js [hunk 2]
```diff
@@ -25,9 +25,15 @@ const redisConfig = {
 	},
 };
 
+const options = {
+	uiConfig: {
+		boardTitle: config.BOARD_TITLE,
+	}
```
Comment (ukiesmack) [inline src/index.js:31]:
```suggestion
	},

```

File: src/index.js [hunk 3]
```diff
@@ -25,9 +25,17 @@ const redisConfig = {
 	},
 };
 
+const options = {
+	uiConfig: {
+		boardTitle: config.BOARD_TITLE,
+	},
+
```
Comment (ukiesmack) [inline src/index.js:32]:
```suggestion
```

File: src/index.js [hunk 4]
```diff
@@ -25,9 +25,16 @@ const redisConfig = {
 	},
 };
 
+const options = {
+	uiConfig: {
+		boardTitle: config.BOARD_TITLE,
+	},
+};
+
```
Comment (ukiesmack) [inline src/index.js:33]:
```suggestion
```

**Comment by haradzienski** [comment]
Hey @Addono, this PR aims to add a pass-through ENV var for the board title - this is useful when you have several environments and want to differentiate between them - would you mind merging this in? Let me know please if something else needs to be updated or added.

**Comment by Addono** [comment]
Hey! Thanks for the contribution!

This was always meant as merely a fork from upstream with newer versions of the deps, but happy to pull the changes in. Upstream seems stale anyway, so why not make this move forward too :)

One change request: Could you update the documentation too 😄  Then I'm happy to merge it 🚀 

> [!NOTE]
> For my own reference, this was the release commit title I was drafting (will end up in the release notes) 😇 
> ```
> feat: allow configuring the board title through env-var `BOARD_TITLE` (#355)
> ```

**Comment by Addono** [comment]
Whatever, I will just do it myself :) Little effort since I can update the PR myself too

**Pull request merged**